main.tf

```diff
@@ -216,3 +216,42 @@ resource "aws_cloudwatch_metric_alarm" "service_not_healthy_alarm_no_lb" {
   threshold           = var.desired_count
   treat_missing_data  = "breaching"
 }
+
+
+resource "aws_cloudwatch_metric_alarm" "service_cpu_utilization_alarm" {
+  alarm_actions       = var.alarm_actions
+  alarm_description   = format("%s service cpu utilization is greater than %s percent of reserved cpu", var.service_name, var.cpu_utilization_alarm_threshold)
+  alarm_name          = format("%s-cpu-utilization-alarm", var.service_name)
+  comparison_operator = "GreaterThanThreshold"
+  dimensions          = {
+    ServiceName = var.service_name
+    ClusterName = var.cluster
+  }
+  evaluation_periods  = 1
```

Comment:
We should make the periods configurable, we probably only care about this if its > 5 periods